package.json

```
@@ -15,10 +15,10 @@
     "dev": "turbo run dev --parallel --continue --filter=next-auth-app... --filter=@auth/core --filter=!./packages/adapter-*",
     "dev-v4:db": "turbo run dev --parallel --continue --filter=next-auth-app-v4...",
     "dev-v4": "turbo run dev --parallel --continue --filter=next-auth-app-v4... --filter=!./packages/adapter-*",
-    "dev:kit": "turbo run dev --parallel --continue --filter=sveltekit-auth-app...",
+    "dev:kit": "turbo run dev --parallel --continue --filter=sveltekit-auth-app... --filter=@auth/core",
```

```suggestion
    "dev:kit": "turbo run dev --parallel --continue --filter=sveltekit-auth-app...
```

This should not be necessary, since `sveltekit-auth-app` depends on `@auth/sveltekit` which depends on `@auth/core`, so it should already be building.